### PR TITLE
fix: livenet , testnet added

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**
Following points have successfully been covered in this PR,
1- Buttons in contest page are similar to home page.
2- price is in the whole number
3- fortmatic is removed
4-  extended contest ticker is displayed if inExtendedContest returns true
5- button is disabled if the withdrawn amount is 0 or user is the part of extended contest
6- mainnet and testnet is dynamically selected according to the chainID.

### CODE REVIEW

File: assets/js/blockchain/vestingContract.js [hunk 1]
```diff
@@ -712,27 +734,40 @@ async function getUserVestingData() {
...
+  if (availableUsdForWithdraw == 0 || inExtendedContest == "true") {
+    const widthdrawBtn = document.getElementById("withdrawBtn");
+    widthdrawBtn.classList.add("is-disabled");
+  }
+  else{
+    const widthdrawBtn = document.getElementById("withdrawBtn");
+    widthdrawBtn.classList.remove("is-disabled");
+  }
+  
+  if(inExtendedContest == true){
```
Comment (bobachef) [inline assets/js/blockchain/vestingContract.js:755]: also you don't need to do this 

```
if(inExtendedContest){ ... 
```
is good enough

Comment (Shaharyar-saleem) [reply assets/js/blockchain/vestingContract.js:755]: done

**Comment by Shaharyar-saleem** [comment]
The following points have been covered:
1- unnecessorory  code is removed from vestingContract.js
2- global and user stats are corrected
3- comma is added

File: assets/js/blockchain/vestingContract.js [hunk 2]
```diff
@@ -610,12 +614,30 @@ const contractAbi = [
...
+  else{
+    vestingContractAddress = liveVestingContract;
+    vestingContract = new ethers.Contract(
+      vestingContractAddress,
+      contractAbi,
+      etherProvider
+    );
+  }
+  return chainID;
+}
```
[the remark this answers is not in the document]
Comment (Shaharyar-saleem) [reply assets/js/blockchain/vestingContract.js]: followed

**Review by bobachef — APPROVED** [review]
good work. just missing the animations but we can do that later.

**Pull request merged**